warehouse/seeds/ntd_agency_to_organization.csv

```diff
@@ -0,0 +1,231 @@
+ntd_id,legacy_ntd_id,organization_name
```

Comment:
Did you guys do any spot checks on this? If we could run a summary on # organizations with GTFS data but no NTD ID, I wonder if we can make a to-do item for someone to check on those. The coverage on this is one of the things I am most nervous about for analyst usage.

Reply:
Sorry I meant more like spot checks on what % of organizations is that, and how many organizations with GTFS don't match, probably most importantly how many organizations in `fct_daily_organization_combined_guideline_checks` don't match

Reply:
https://github.com/cal-itp/data-infra/issues/2272